tasks/main.yml

```diff
@@ -77,5 +77,20 @@
     name: "{{ item }}"
     state: started
     enabled: yes
+    use: "service"
   with_items: "{{ varnish_enabled_services | default([]) }}"
-  when: varnish_enabled_services
+  when: >
+    varnish_enabled_services and
+    (ansible_os_family != 'Debian' and ansible_distribution_release != "xenial")
+
+# See: https://github.com/ansible/ansible/issues/22303
+- name: Ensure Varnish services are started enabled on startup (Xenial specific)
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+    use: "service"
+  with_items: "{{ varnish_enabled_services | default([]) }}"
+  when: >
+    varnish_enabled_services and
```

Comment:
For compound `when` conditions, you can use the style:

```yaml
  when:
    - varnish_enabled_services
    - ansible_os_family == 'Debian' and ansible_distribution_release == "xenial"
```

I just discovered this recently, and am trying to get all my roles to use that style for complex conditionals; can you update these to use that style too?